app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.split() != 3:
```

Comment:
The condition `command.split() != 3` is incorrect. `command.split()` returns a list, and you should compare its length to 3 using `len(command.split()) != 3`.